src/commands/periscope/helpers/periscopehelper.ts

```diff
@@ -0,0 +1,265 @@
+                if (!name) {
+                    vscode.window.showInformationMessage(`Storage Id is malformed: ${item.storageAccountId}`);
+                    return;
+                }
+                storageAccountNameToStorageIdArray.push({ id: item.storageAccountId, label: name });
+            }
+        });
+
+        // accounts is now an array of {id, name}
+        const accountQuickPicks = storageAccountNameToStorageIdArray;
+
+        // Create quick pick for more than 1 storage account scenario.
+        const selectedQuickPick = await vscode.window.showQuickPick(
+            accountQuickPicks,
+            {
+                placeHolder: "Select storage account for Periscope deployment:",
+                ignoreFocusOut: true,
+                onDidSelectItem: (item: string) => { return item.toString(); }
+            });
+
+        if (selectedQuickPick?.label) {
+            return selectedQuickPick?.id;
+        }
+
+    } else if (diagnosticSettings && diagnosticSettings.value!.length === 1) {
+        // In case of only one storage account associated, use the one (1) as default storage account and no UI will be displayed.
+        const selectedStorageAccount = diagnosticSettings.value![0].storageAccountId!;
+        return selectedStorageAccount;
+    }
+
+    return undefined;
+}
+
+export async function getStorageInfo(
+    cluster: AksClusterTreeItem,
+    diagnosticStorageAccountId: string
+): Promise<PeriscopeStorage | undefined> {
+    try {
+        const { resourceGroupName, name: accountName } = parseResource(diagnosticStorageAccountId!);
+
+        if (!resourceGroupName || !accountName) {
+            vscode.window.showErrorMessage(`Invalid storage id ${diagnosticStorageAccountId} associated with the cluster`);
+            return;
+        }
+
+        // Get keys from storage client.
+        const storageClient = new ast.StorageManagementClient(cluster.root.credentials, cluster.root.subscriptionId);
+        const storageAccKeyList = await storageClient.storageAccounts.listKeys(resourceGroupName, accountName);
+        const storageKey = storageAccKeyList.keys?.find((it) => it.keyName === "key1")?.value!;
+
+        const clusterStorageInfo = {
+            storageName: accountName,
+            storageKey: storageKey,
+            storageDeploymentSas: getSASKey(accountName, storageKey, LinkDuration.DownloadNow),
+            sevenDaysSasKey: getSASKey(accountName, storageKey, LinkDuration.Shareable)
+        };
+
+        return clusterStorageInfo;
+    } catch (e) {
+        vscode.window.showErrorMessage(`Storage associated with cluster had following error: ${e}`);
+        return undefined;
+    }
+}
+
+export async function prepareAKSPeriscopeDeploymetFile(
+    clusterStorageInfo: PeriscopeStorage
+): Promise<string | undefined> {
+    const tempFile = tmp.fileSync({ prefix: "aks-periscope-", postfix: `.yaml` });
+    const axios = require("axios");
```

Comment:
Again, is there a reason why you don't `import * as axios from 'axios';` this at the top of the file?